base/error.jl

```diff
@@ -227,11 +247,16 @@ ERROR: AssertionError: 3 is an odd number!
 julia> @assert isodd(3) "What even are numbers?"
 ```
 """
-macro assert(ex, msgs...)
-    msg = isempty(msgs) ? ex : msgs[1]
+macro assert(ex, args...)
+
+    aargs = filter(e -> !Meta.isexpr(e, :(=)), args)
+    aakws = filter(e -> Meta.isexpr(e, :(=)), args)
```

Comment:
`Meta` is defined much later in the bootstrapping. Luckily, `Base.Meta.isexpr === Base.isexpr`, and `isexpr` is already defined.

```suggestion
    aargs = filter(e -> !isexpr(e, :(=)), args)
    aakws = filter(e -> isexpr(e, :(=)), args)
```